Sound/Tidal/Stream.hs

```diff
@@ -218,6 +218,17 @@ infixl 1 |/|
 (|/|) :: ParamPattern -> ParamPattern -> ParamPattern
 (|/|) = mergeNumWith (div) (/)
 
+{- | These are shorthand for merging lists of patterns with @#@, @|*|@, @|+|@,
```

Comment:
There might be a higher-level way of defining it more simply, but I couldn't figure it out.

I used three across the board for consistency, and because `**` is already used for exponentiation and `++` for list concatenation.

Really, these only exist because I got tired of typing long compositions of sections, like
`every 2 ((|*| speed "2") . (|*| end "0.5") . (|*| shape "2"))`, and this was the only thing I could come up with that avoided paren-matching hell and didn't involve lots of extra keystrokes.